src/main/utils/log.ts

```diff
@@ -12,14 +12,14 @@ export const Log =
     descriptor.value = async function (...args: any[]) {
       const startTime = Date.now()
 
-      _logger.verbose(
-        `Method ${propertyKey} called with arguments: ${args.map((it) => `${it}`).join(', ')}`
-      )
+      // _logger.verbose(
+      //   `Method ${propertyKey} called with arguments: ${args.map((it) => `${it}`).join(', ')}`
+      // )
 
       try {
         const result = await originalMethod.apply(this, args)
-        const endTime = Date.now()
-        _logger.verbose(`Method ${propertyKey} executed successfully in ${endTime - startTime}ms`)
+        // const endTime = Date.now()
+        // _logger.verbose(`Method ${propertyKey} executed successfully in ${endTime - startTime}ms`)
```

Comment:
Not a fan of commented out code...

Reply:
Id rather it be commented out in here than re-typed every time I want to debug something low level